src/client/components/admin/UserContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import Users from './Users';
+import UserTableRow from './UserTableRow';
+
+class UserContainer extends React.Component< {}, {
+    userData: Array<React.ReactElement<HTMLDivElement>>} > {
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            userData: [],
+        };
+    }
+
+    public async componentDidMount(): Promise<void> {
+        const result = await fetch('/graphql', {
+            credentials: 'same-origin',
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                query: '{users { user_id, name, email, user_class }}',
+            }),
+        });
+        const data = await result.json();
+        const users = data.data.users;
+        const newUserData: Array<React.ReactElement<HTMLDivElement>> = [];
+        for (let i = 0; i < users.length; i++) {
+            newUserData.push(<UserTableRow
```

Comment:
I'm not sure if this UI knowledge should be here.. Store the userData as json and build UI in the presentational component